### ISSUE TITLE
Enchantments no longer work (Paper)

### ISSUE THREAD

**Issue by aurel85** [Jikoo/EnchantableBlocks#27]
Hi,

For a few days now, the enchantments on the old ovens as on the new ones no longer work at all for no particular reason. The only thing that could have interfered with the plugin is I think the latest updates from Paper.
The problem must therefore lie between versions 700-750 because I updated yesterday from the 735 to 750 but the problems started before.

I am using Paper 1.16.5 and the latest version of the plugin (2.0.7).

![Capture](https://user-images.githubusercontent.com/24486147/120179003-5e26e180-c20a-11eb-9907-c64d60021d8a.PNG)

**Comment by Jikoo** [comment]
So when you say that enchantments don't work, is it all enchantments or just specific ones? Fortune, silk touch, and unbreaking all are Bukkit API-only stuff. Efficiency I could see breaking, though Paper's usually careful enough about how they do their changes that it generally doesn't.

**Comment by aurel85** [comment]
Here is a video to better understand : https://www.youtube.com/watch?v=WB7UySJNY70

And i just saw this error in the console : https://pastebin.com/UUCUNdBU

**Comment by Jikoo** [comment]
Ok, that's a problem with a stonecutting recipe registered to the server. That means that anything that attempts to use the recipe iterator on your server will break because the stonecutter recipe somehow doesn't have a valid output. You need to track down and remove the invalid recipe.

**Comment by aurel85** [comment]
I found the culprit it is not Paper but this datapack which I added very recently. When I remove it everything works normally.

https://www.planetminecraft.com/data-pack/stone-and-wood-cutter/

i dont know what can be done at your level or datapack creator level or even if i can fix this on my own. Note that I deleted some crafts in the datapack because they wanted to give items that do not exist in the game so it crashed the players.
